--- unixODBC-2.1.1/Makefile.am.orig	Tue Dec  4 11:35:04 2001
+++ unixODBC-2.1.1/Makefile.am	Fri Jan 18 00:18:41 2002
@@ -125,7 +125,7 @@
 	vmsbuild.com
 
 install-data-am:
-	touch ${sysconfdir}/odbcinst.ini
-	touch ${sysconfdir}/odbc.ini
-	mkdir -p ${sysconfdir}/ODBCDataSources
+	mkdir -p $(DESTDIR)$(sysconfdir)/ODBCDataSources
+	touch $(DESTDIR)$(sysconfdir)/odbcinst.ini
+	touch $(DESTDIR)$(sysconfdir)/odbc.ini
 
