diff -ur unixODBC-2.2.9.orig/include/sqltypes.h unixODBC-2.2.9/include/sqltypes.h
--- unixODBC-2.2.9.orig/include/sqltypes.h	2004-06-25 19:01:15.000000000 +0200
+++ unixODBC-2.2.9/include/sqltypes.h	2004-10-03 03:35:55.969613088 +0200
@@ -256,9 +256,14 @@
 typedef signed short int        SWORD;
 typedef unsigned short int      UWORD;
 typedef unsigned int            UINT;
+#if (SIZEOF_LONG == 4)
 typedef signed long             SLONG;
-typedef signed short            SSHORT;
 typedef unsigned long           ULONG;
+#else
+typedef signed int              SLONG;
+typedef unsigned int            ULONG;
+#endif
+typedef signed short            SSHORT;
 typedef unsigned short          USHORT;
 typedef double                  SDOUBLE;
 typedef double            		LDOUBLE;
