--- unixODBC-2.2.6/gODBCConfig/src/Makefile.am.orig	2001-10-17 18:40:23.000000000 +0200
+++ unixODBC-2.2.6/gODBCConfig/src/Makefile.am	2003-07-11 21:57:30.000000000 +0200
@@ -25,7 +25,7 @@
 gODBCConfig_LDADD = $(GNOME_LIBDIR) \
 		$(GNOMEUI_LIBS) \
 		$(INTLLIBS) \
-		-L$(ODBCLIBS) $(ODBCLIB) \
+		../../odbcinst/libodbcinst.la \
 		libgtkodbcconfig.la 
 		
 
--- unixODBC-2.2.6/gODBCConfig/Makefile.am.orig	2002-12-16 19:54:22.000000000 +0100
+++ unixODBC-2.2.6/gODBCConfig/Makefile.am	2003-07-11 22:01:13.000000000 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = m4  po macros src
+SUBDIRS = po src
 
 install-data-local:
 	@$(NORMAL_INSTALL)
--- unixODBC-2.2.6/gODBCConfig/configure.in.orig	2002-12-16 19:54:27.000000000 +0100
+++ unixODBC-2.2.6/gODBCConfig/configure.in	2003-07-11 22:01:23.000000000 +0200
@@ -45,33 +45,9 @@
 fi
 ])
 odbcdir=""
-AC_MSG_CHECKING(for ODBC files)
-for d in $dir /usr /usr/local
-do
-	if test -f $d/lib/libodbcinst.so 
-	then
-        AC_MSG_RESULT(found ODBC in $d)
-        ODBCLIB="-lodbcinst"
-        odbcdir=$d
-        break
-	fi
-done
-if test x$odbcdir = x
-then
-	AC_MSG_ERROR(ODBC backend not found)
-else
-	AC_DEFINE(HAVE_ODBC)
-	ODBCINCLUDES=${odbcdir}/include
-	ODBCLIBS=${odbcdir}/lib
-fi
-
-AC_SUBST(ODBCINCLUDES)
-AC_SUBST(ODBCLIBS)
-AC_SUBST(ODBCLIB)
 
-AC_OUTPUT([ m4/Makefile 
+AC_OUTPUT([
 Makefile
-macros/Makefile
 src/Makefile
 po/Makefile.in
 ])
