--- unixODBC-2.2.9/include/sqltypes.h.orig	2004-06-25 19:01:15.000000000 +0200
+++ unixODBC-2.2.9/include/sqltypes.h	2004-07-07 21:30:56.439338072 +0200
@@ -48,7 +48,7 @@
  */
 
 #ifndef SIZEOF_LONG
-# if defined(__alpha) || defined(__sparcv9) || defined(__LP64__) || || (defined(__HOS_AIX__) && defined(_LP64))
+# if defined(__alpha) || defined(__sparcv9) || defined(__LP64__) || (defined(__HOS_AIX__) && defined(_LP64))
 # define SIZEOF_LONG        8
 #else
 # define SIZEOF_LONG        4
