--- unixODBC-2.2.11/sqp/sqpStoreDataType.c.orig	2004-07-26 10:34:47.000000000 +0200
+++ unixODBC-2.2.11/sqp/sqpStoreDataType.c	2006-08-27 18:23:27.375835500 +0200
@@ -9,6 +9,13 @@
 #include "sqp.h"
 #include <sqlext.h>
 
+#ifndef WIN32
+#ifndef HAVE_STRICMP
+#define stricmp(s1,s2) 		strcasecmp(s1,s2)
+#define strnicmp(s1,s2,n)	strncasecmp(s1,s2,n)
+#endif
+#endif
+
 void sqpStoreDataType( char *pszType, int nPrecision, int nScale )
 {
     HSQPDATATYPE    hDataType;
--- unixODBC-2.2.12/gODBCConfig/src/callbacks.c.orig	2006-04-18 12:29:28.000000000 +0200
+++ unixODBC-2.2.12/gODBCConfig/src/callbacks.c	2006-11-24 17:44:31.329505000 +0100
@@ -13,6 +13,13 @@
 #include "interface.h"
 #include "support.h"
 
+#ifndef WIN32
+#ifndef HAVE_STRICMP
+#define stricmp(s1,s2) 		strcasecmp(s1,s2)
+#define strnicmp(s1,s2,n)	strncasecmp(s1,s2,n)
+#endif
+#endif
+
 static char add_driver_name[ 257 ];
 static char add_driver_description[ 257 ];
 static char add_driver_driver[ 257 ];
