--- unixODBC-2.2.9/Makefile.am.orig	2004-06-29 14:11:02.000000000 +0200
+++ unixODBC-2.2.9/Makefile.am	2004-07-07 20:10:49.461109840 +0200
@@ -3,7 +3,6 @@
     log \
 	lst \
 	ini \
-    	libltdl \
 	odbcinst \
 	DriverManager \
 	odbcinstQ \
--- unixODBC-2.2.9/configure.in.orig	2004-07-07 20:09:10.724120144 +0200
+++ unixODBC-2.2.9/configure.in	2004-07-07 20:29:03.316818496 +0200
@@ -135,15 +135,13 @@
 AC_MSG_CHECKING( Have we enabled using RTDL_GROUP )
 if test "x$rtldgroup" = "xtrue"; then
 AC_MSG_RESULT( yes )
-AC_LIBLTDL_CONVENIENCE
 else
 AC_MSG_RESULT( no );
-AC_LIBLTDL_CONVENIENCE_G
 fi
+AC_LIBLTDL_INSTALLABLE
 
 dnl Check for dlopen support
 AC_LIBTOOL_DLOPEN
-AC_CONFIG_SUBDIRS(libltdl)
 
 dnl Configure libtool
 AM_PROG_LIBTOOL
