diff -ur unixODBC-2.0.8.org/Drivers/MiniSQL/Makefile.am unixODBC-2.0.8/Drivers/MiniSQL/Makefile.am
--- unixODBC-2.0.8.org/Drivers/MiniSQL/Makefile.am	Mon Oct 15 15:14:52 2001
+++ unixODBC-2.0.8/Drivers/MiniSQL/Makefile.am	Mon Oct 15 15:37:14 2001
@@ -3,7 +3,7 @@
 
 INCLUDES = -I../../include -I. -I@msql_headers@ -I${INCLTDL}
 
-DEFS = -DUNIXODBC @DEFS@ -DDRIVER_NAME=\"odbcminisql\"
+DEFS_OUR = -DUNIXODBC -DDRIVER_NAME=\"odbcminisql\"
 
 libodbcmini_la_SOURCES = \
 	SQLAllocConnect.c \
@@ -198,3 +198,4 @@
 libodbcmini_la_LDFLAGS = -version-info 1:0:0 \
 	-L@msql_libraries@ -lmsql -module
 
+DEFS = @DEFS@ $(DEFS_OUR)
\ No newline at end of file
Only in unixODBC-2.0.8/Drivers/MiniSQL: Makefile.am~
diff -ur unixODBC-2.0.8.org/exe/Makefile.am unixODBC-2.0.8/exe/Makefile.am
--- unixODBC-2.0.8.org/exe/Makefile.am	Mon Oct 15 15:14:50 2001
+++ unixODBC-2.0.8/exe/Makefile.am	Mon Oct 15 15:44:01 2001
@@ -4,10 +4,6 @@
 
 EXTRA_DIST = isql.h
 
-odbcinst_DEPENDENCIES = \
-    $(LIBLTDL) \
-    ../odbcinst/libodbcinst.la
-
 odbcinst_LDADD = \
 	../odbcinst/libodbcinst.la \
 	$(LIBLTDL) $(LIBADD_DL) 
Only in unixODBC-2.0.8/exe: Makefile.am~
