--- unixODBC-2.2.11/gODBCConfig/src/Makefile.am.orig	2004-09-29 16:23:50.000000000 +0200
+++ unixODBC-2.2.11/gODBCConfig/src/Makefile.am	2005-03-27 12:45:46.264419832 +0200
@@ -21,13 +21,14 @@
 	support.c support.h \
 	interface.h \
 	callbacks.c callbacks.h
+libgtkodbcconfig_la_LIBADD = \
+		$(GNOME_LIBDIR) $(GNOMEUI_LIBS) \
+		../../odbcinst/libodbcinst.la ../../ini/libinilc.la
 
 gODBCConfig_DEPENDENCIES = libgtkodbcconfig.la
 
-gODBCConfig_LDADD = $(GNOME_LIBDIR) \
-		$(GNOMEUI_LIBS) \
+gODBCConfig_LDADD = \
 		$(INTLLIBS) \
-		-L$(ODBCLIBS) $(ODBCLIB) \
 		libgtkodbcconfig.la 
 		
 
--- unixODBC-2.2.6/gODBCConfig/configure.in.orig	2003-07-21 12:56:54.000000000 +0200
+++ unixODBC-2.2.6/gODBCConfig/configure.in	2003-08-09 10:37:38.000000000 +0200
@@ -45,29 +45,6 @@
 fi
 ])
 odbcdir=""
-AC_MSG_CHECKING(for ODBC files)
-for d in $dir /usr /usr/local
-do
-	if test -f $d/lib/libodbcinst.so 
-	then
-        AC_MSG_RESULT(found ODBC in $d)
-        ODBCLIB="-lodbcinst"
-        odbcdir=$d
-        break
-	fi
-done
-if test x$odbcdir = x
-then
-	AC_MSG_ERROR(ODBC backend not found)
-else
-	AC_DEFINE(HAVE_ODBC)
-	ODBCINCLUDES=${odbcdir}/include
-	ODBCLIBS=${odbcdir}/lib
-fi
-
-AC_SUBST(ODBCINCLUDES)
-AC_SUBST(ODBCLIBS)
-AC_SUBST(ODBCLIB)
 
 dnl Keep autoheader happy
 AC_DEFINE([ENABLE_NLS], [], [Description])
@@ -79,7 +56,6 @@
 AC_DEFINE([PACKAGE_LOCALE_DIR], [], [Description])
 
 AC_OUTPUT([ Makefile 
-macros/Makefile
 src/Makefile
 po/Makefile.in
 ])
