--- unixODBC-2.1.1/configure.in~	Fri Jan 18 02:36:49 2002
+++ unixODBC-2.1.1/configure.in	Fri Jan 18 02:36:49 2002
@@ -273,7 +273,7 @@
 dnl I need to check for libsocket before testing X
 AC_CHECK_LIB(socket, socket, [LIBSOCKET="-lsocket"]) dnl for Solaris
 AC_SUBST(LIBSOCKET)
-AC_CHECK_LIB(nsl, gethostbyname, [LIBNSL="-lnsl"]) dnl for Solaris
+AC_CHECK_FUNC(gethostbyname, [], AC_CHECK_LIB(nsl, gethostbyname, [LIBNSL="-lnsl"]))
 AC_SUBST(LIBNSL)
 
 if test "x$macosx" = "xyes"; then
