diff -Nur unixODBC-2.2.4.orig/acinclude.m4 unixODBC-2.2.4/acinclude.m4
--- unixODBC-2.2.4.orig/acinclude.m4	Fri Jan  3 17:04:44 2003
+++ unixODBC-2.2.4/acinclude.m4	Fri Jan 24 20:15:02 2003
@@ -1268,7 +1268,7 @@
 dnl which includes <sys/select.h> which contains a prototype for
 dnl select.  Similarly for bzero.
 [#include <assert.h>
-]ifelse(AC_LANG, CPLUSPLUS, [#ifdef __cplusplus
+#ifdef __cplusplus
 extern "C"
 #endif
 ])dnl
