diff -Nru unixODBC-2.0.3/acinclude.m4 unixODBC-2.0.3.new/acinclude.m4
--- unixODBC-2.0.3/acinclude.m4	Sun Jan 14 21:44:57 2001
+++ unixODBC-2.0.3.new/acinclude.m4	Mon Jan 15 01:56:30 2001
@@ -3166,10 +3166,6 @@
 enable_shared=AC_ENABLE_SHARED_DEFAULT)dnl
 ])
 
-# AC_DISABLE_SHARED - set the default shared flag to --disable-shared
-AC_DEFUN(AC_DISABLE_SHARED, [AC_BEFORE([$0],[AC_LIBTOOL_SETUP])dnl
-AC_ENABLE_SHARED(no)])
-
 # AC_ENABLE_STATIC - implement the --enable-static flag
 # Usage: AC_ENABLE_STATIC[(DEFAULT)]
 #   Where DEFAULT is either `yes' or `no'.  If omitted, it defaults to
@@ -3388,35 +3384,6 @@
   esac
   LIBLTDL=ifelse($#,1,$1,['${top_builddir}/libltdl'])/libltdlc.la
   INCLTDL=ifelse($#,1,-I$1,['-I${top_builddir}/libltdl'])
-])
-
-# AC_LIBLTDL_INSTALLABLE[(dir)] - sets LIBLTDL to the link flags for
-# the libltdl installable library, and adds --enable-ltdl-install to
-# the configure arguments.  Note that LIBLTDL is not AC_SUBSTed, nor
-# is AC_CONFIG_SUBDIRS called.  If DIR is not provided, it is assumed
-# to be `${top_builddir}/libltdl'.  Make sure you start DIR with
-# '${top_builddir}/' (note the single quotes!) if your package is not
-# flat, and, if you're not using automake, define top_builddir as
-# appropriate in the Makefiles.
-# In the future, this macro may have to be called after AC_PROG_LIBTOOL.
-AC_DEFUN(AC_LIBLTDL_INSTALLABLE, [AC_BEFORE([$0],[AC_LIBTOOL_SETUP])dnl
-  AC_CHECK_LIB(ltdl, main,
-  [test x"$enable_ltdl_install" != xyes && enable_ltdl_install=no],
-  [if test x"$enable_ltdl_install" = xno; then
-     AC_MSG_WARN([libltdl not installed, but installation disabled])
-   else
-     enable_ltdl_install=yes
-   fi
-  ])
-  if test x"$enable_ltdl_install" = x"yes"; then
-    ac_configure_args="$ac_configure_args --enable-ltdl-install"
-    LIBLTDL=ifelse($#,1,$1,['${top_builddir}/libltdl'])/libltdl.la
-    INCLTDL=ifelse($#,1,-I$1,['-I${top_builddir}/libltdl'])
-  else
-    ac_configure_args="$ac_configure_args --enable-ltdl-install=no"
-    LIBLTDL="-lltdl"
-    INCLTDL=
-  fi
 ])
 
 dnl old names
diff -Nru unixODBC-2.0.3/configure.in unixODBC-2.0.3.new/configure.in
--- unixODBC-2.0.3/configure.in	Sun Jan 14 21:44:57 2001
+++ unixODBC-2.0.3.new/configure.in	Mon Jan 15 02:00:51 2001
@@ -189,8 +189,6 @@
 
 AC_CHECK_HEADERS(malloc.h unistd.h pwd.h crypt.h limits.h synch.h strings.h string.h locale.h)
 
-INCLUDES="$INCLUDES $USER_INCLUDES";
-
 dnl only build the mSQL code if the headers are in place
 AC_CHECK_HEADERS(msql.h,[msql=true],
 [
@@ -200,12 +198,11 @@
     AC_CHECK_HEADERS( $ac_dir/msql.h,
     [
         msql=true;
-        INCLUDES="$INCLUDES $USER_INCLUDES -I$ac_dir";
+        INCLUDES="$INCLUDES -I$ac_dir";
     ])
    done
 ])
 
-AC_SUBST(INCLUDES)
 dnl AC_SUBST(all_includes)
 dnl AC_SUBST(all_libraries)
 
