--- unixODBC-2.2.12/gODBCConfig/src/Makefile.am.orig	2005-09-17 15:08:16.000000000 +0200
+++ unixODBC-2.2.12/gODBCConfig/src/Makefile.am	2006-11-24 14:12:29.549505000 +0100
@@ -21,13 +21,12 @@
 	support.c support.h \
 	interface.h \
 	callbacks.c callbacks.h
+libgtkodbcconfig_la_LIBADD = \
+		$(GNOME_LIBDIR) $(GNOMEUI_LIBS) \
+		../../odbcinst/libodbcinst.la ../../ini/libinilc.la
 
 gODBCConfig_DEPENDENCIES = libgtkodbcconfig.la ../../ini/libinilc.la
 
-gODBCConfig_LDADD = $(GNOME_LIBDIR) \
-		$(GNOMEUI_LIBS) \
+gODBCConfig_LDADD = \
 		$(INTLLIBS) \
-		-L$(ODBCLIBS) $(ODBCLIB) \
-		libgtkodbcconfig.la \
-		../../odbcinst/libodbcinst.la \
-		../../ini/libinilc.la
+		libgtkodbcconfig.la
--- unixODBC-2.2.12/gODBCConfig/configure.in.orig	2005-09-16 20:09:56.000000000 +0200
+++ unixODBC-2.2.12/gODBCConfig/configure.in	2006-11-24 15:23:16.209505000 +0100
@@ -21,7 +21,7 @@
 dnl Add the languages which your application supports here.
 ALL_LINGUAS=""
 AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION(0.11.4
+AM_GNU_GETTEXT_VERSION(0.11.4)
 
 dnl Set PACKAGE_LOCALE_DIR in config.h.
 if test "x${prefix}" = "xNONE"; then
@@ -47,29 +47,6 @@
 fi
 ])
 odbcdir=""
-AC_MSG_CHECKING(for ODBC files)
-for d in $dir /usr /usr/local
-do
-	if test -f $d/lib/libodbcinst.so 
-	then
-        AC_MSG_RESULT(found ODBC in $d)
-        ODBCLIB="-lodbcinst"
-        odbcdir=$d
-        break
-	fi
-done
-if test x$odbcdir = x
-then
-	AC_MSG_ERROR(ODBC backend not found)
-else
-	AC_DEFINE(HAVE_ODBC)
-	ODBCINCLUDES=${odbcdir}/include
-	ODBCLIBS=${odbcdir}/lib
-fi
-
-AC_SUBST(ODBCINCLUDES)
-AC_SUBST(ODBCLIBS)
-AC_SUBST(ODBCLIB)
 
 dnl Keep autoheader happy
 AC_DEFINE([ENABLE_NLS], [], [Description])
@@ -81,8 +58,7 @@
 AC_DEFINE([PACKAGE_LOCALE_DIR], [], [Description])
 
 AC_OUTPUT([ Makefile 
 macros/Makefile
 src/Makefile
-po/Makefile.in
 ])
 
